lib/cassandra/executors.rb

```diff
@@ -80,6 +80,7 @@ def run
 
         loop do
           tasks = nil
+          @term ||= nil
```

Comment:
@avalanche123 I saw that, but I still get the warning as a consumer. "Something something threading something"?